### ISSUE TITLE
Spectre's Shadowblade

### ISSUE THREAD

**Issue by Shin-Hayata1** [project-bo4/shield-development#56]
I've come across an issue where when using Spectre's Shadowblade the roll button does not function on controller.

**Comment by project-bo4** [comment]
rolling should be performed by circle correct? other than that does it works fine on everything else?

**Comment by Shin-Hayata1** [comment]
Yeah its done by using circle, or in this case B, because I'm using an Xbox Controller. From what I've tried it doesn't work on my keyboard nor my controller. It simply just doesn't work.

**Comment by project-bo4** [comment]
Take a backup from players folder and delete it completely so all bindings and settings resets and try to see if it fixes. report back if it does so.
